.common/templates/lib/controller/_container.tpl

```diff
@@ -34,9 +34,7 @@ The main container included in the controller.
   {{- end }}
   {{- end }}
   {{- end }}
-  {{- if .Values.appVolumeMounts }}
-    {{- toYaml .Values.appVolumeMounts | nindent 2 }}
-  {{- end }}
+  {{- include "configuredAppVolumeMounts" . | nindent 2 }}
```

Comment:
What do you think about removing the `\n` from `nindent 2`?

Maybe just use `indent 2` instead? Visually, I think it looks better when viewing it rendered.